dbms/src/Common/iostream_debug_helpers.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+#include <iostream>
+#include <common/iostream_debug_helpers.h>
+
+
+#include <DataStreams/IBlockInputStream.h>
+std::ostream & operator<<(std::ostream & stream, const DB::IBlockInputStream & what)
```

Comment:
Function defined in header file - this will lead to multiple definitions error when linking.